src/s_audio.c

```diff
@@ -65,7 +65,7 @@ static int audio_naudiooutdev = -1;
 static int audio_audiooutdev[MAXAUDIOOUTDEV];
 static int audio_audiochoutdev[MAXAUDIOOUTDEV];
 static char audio_outdevnames[MAXMIDIINDEV * DEVDESCSIZE];
-static int audio_rate;
+static int audio_rate = 0;
 static int audio_advance = -1;
 static int audio_callback;
 static int audio_blocksize;
```

Comment:
Probably init to 0 so it will be set in `sys_get_audio_params`.

Reply:
actually, `static int` values without an explicit value will always be initialized to **0** in C.

Reply:
Ok, although I always like to be explicit when it's something expected/important. :)